examples/6field-simple/elm_6f.cxx

```diff
@@ -12,1677 +12,1682 @@
+    hp_length = options["hp_length"].withDefault(0.04);
+
+    // sink factor in pressure
+    // sink in pressure
+    sink_vp = options["sink_vp"].withDefault(-1.0);
+    // the percentage of radial grid points for sink profile radial
+    // width in pressure
+    sp_width = options["sp_width"].withDefault(0.05);
+    // the percentage of radial grid points for sink profile radial
+    // domain in pressure
+    sp_length = options["sp_length"].withDefault(0.04);
+
+    // left edge sink factor in vorticity
+    // left edge sink in vorticity
+    sink_Ul = options["sink_Ul"].withDefault(-1.0);
+    // the percentage of left edge radial grid points for sink profile
+    // radial width in vorticity
+    su_widthl = options["su_widthl"].withDefault(0.06);
+    // the percentage of left edge radial grid points for sink profile
+    // radial domain in vorticity
+    su_lengthl = options["su_lengthl"].withDefault(0.15);
+
+    // right edge sink factor in vorticity
+    // right edge sink in vorticity
+    sink_Ur = options["sink_Ur"].withDefault(-1.0);
+    // the percentage of right edge radial grid points for sink profile
+    // radial width in vorticity
+    su_widthr = options["su_widthr"].withDefault(0.06);
+    // the percentage of right edge radial grid points for sink profile
+    // radial domain in vorticity
+    su_lengthr = options["su_lengthr"].withDefault(0.15);
+
+    // Compressional terms
+    phi_curv = options["phi_curv"].withDefault(true);
+    g = options["gamma"].withDefault(5.0 / 3.0);
+
+    if (!include_curvature)
+      b0xcv = 0.0;
+
+    if (!include_jpar0)
+      J0 = 0.0;
+
+    if (noshear) {
+      if (include_curvature)
+        b0xcv.z += I * b0xcv.x;
+      I = 0.0;
+    }
 
-  Tau_ie = Tibar / Tebar;
+    //////////////////////////////////////////////////////////////
+    // SHIFTED RADIAL COORDINATES
 
-  Va = sqrt(Bbar * Bbar / (MU0 * Mi * Nbar * density));
+    if (mesh->IncIntShear) {
+      // BOUT-06 style, using d/dx = d/dpsi + I * d/dz
+      coord->IntShiftTorsion = I;
 
-  Tbar = Lbar / Va;
+    } else {
+      // Dimits style, using local coordinate system
+      if (include_curvature)
+        b0xcv.z += I * b0xcv.x;
+      I = 0.0; // I disappears from metric
+    }
 
-  output.write("Normalisations: Bbar = {:e} T   Lbar = {:e} m\n", Bbar, Lbar);
-  output.write("                Va = {:e} m/s   Tbar = {:e} s\n", Va, Tbar);
-  output.write("                Nbar = {:e} * {:e} m^-3\n", Nbar, density);
-  output.write("Tibar = {:e} eV   Tebar = {:e} eV    Ti/Te = {:e}\n", Tibar, Tebar, Tau_ie);
-  output.write("    Resistivity\n");
+    //////////////////////////////////////////////////////////////
+    // NORMALISE QUANTITIES
 
-  Upara0 = KB * Tebar * eV_K / (Zi * ee * Bbar * Va * Lbar);
-  Upara1 = KB * Tebar * eV_K / Mi / Va / Va;
-  output.write("vorticity cinstant: Upara0 = {:e}     Upara1 = {:e}\n", Upara0, Upara1);
+    if (mesh->get(Bbar, "bmag")) // Typical magnetic field
```

Comment:
warning: statement should be inside braces [readability-braces-around-statements]

```cpp
if (mesh->get(Bbar, "bmag")) // Typical magnetic field
                            ^
                             {
```